pymc3/distributions/continuous.py

```diff
@@ -2723,6 +2717,21 @@ def logcdf(value, alpha, beta):
         )
 
 
+class HalfStudentTRV(RandomVariable):
+    name = "halfstudentt"
+    ndim_supp = 0
+    ndims_params = [0, 0, 0]
```

Comment:
`ndims_params` should be `[0, 0]`, i.e., the same length as the number of parameters